virttest/error_message_bus.py

```diff
@@ -0,0 +1,25 @@
+"""
+Global background error event bus for vt test object.
+This aims to share error event bus with asynchronous tasks launched by
+avocado-vt, so that those tasks that have no reference to test could push error
+to error the test.
+
+IMPORTANT: only for internal use inside avocado-vt.
+"""
+
+from six.moves import queue
+
+
+error_message_bus = queue.Queue()
+
+
+def clear_bus():
```

Comment:
I think `def clear_bus()` the function's name is little bit confusing  for both clear and retrieve errors.

Might be better we can separate it as
```python
def clear_bus():
    with error_message_bus.mutex:
        error_message_bus.queue.clear()
```
The current `def clear_bus():` would be `retrieve errors`.

Reply:
It is not enough just call `queue.clear()` to clear all items enqueued. The safest way to do this is to pop out each item. There is no need to define two functions doing basically the same thing. Maybe rename it will be a good approach.